src/app/connections/list/list.component.html

```diff
@@ -62,42 +62,41 @@ <h4 class="modal-title pull-left">Confirm Delete</h4>
             <!-- Card Heading -->
 
             <div class="card-pf-heading-kebab">
-
               <!-- Card Actions -->
-
-              <div *ngIf="showKebab"
-                   dropdown
-                   class="dropdown dropdown-kebab-pf pull-right"
-                   container="body">
-                <button dropdownToggle
-                        class="btn btn-link dropdown-toggle"
-                        type="button"
-                        id="dropdownKebabRight9">
+              <div *ngIf="showKebab">
+                <div dropdown
+                     class="dropdown dropdown-kebab-pf pull-right">
+                  <button dropdownToggle
+                          class="btn btn-link dropdown-toggle"
+                          type="button"
+                          id="dropdownKebabRight9"
+                          (click)="$event.stopPropagation()">
                   <span class="fa fa-ellipsis-v"></span>
                 </button>
-                <ul *dropdownMenu
-                    class="dropdown-menu dropdown-menu-right"
-                    role="menu">
-                  <li role="menuitem">
-                    <a [routerLink]="['/connections', connection.id]"
-                       class="dropdown-item">View</a>
-                  </li>
-                  <!--
+                  <ul *dropdownMenu
```

Comment:
Sorry, I'm sure we keep going back and forth on this without doing it on purpose lol. We should probably share IDE-specific settings & general style guides.